force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -2,11 +2,15 @@
     <template if:true={currentUserHasAccess}>
         <div class="slds-grid slds-theme_default eda-height_full">
             <div
-                class="slds-col slds-p-top_large slds-p-left_small slds-text-heading_small slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full"
+                class="slds-col slds-p-top_x-small slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full"
+                data-qa-locator="edaSettingsNavigation"
             >
-                {labelReference.settingsNavigation}
+                <c-eda-settings-navigation onsettingsnavigation={handleSettingsNavigation}></c-eda-settings-navigation>
             </div>
-            <div class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full">
+            <div
+                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full"
```

Comment:
For the 2nd column do we need unified padding? It looks like we are relying on child components to handle paddings, which can be hard to manage.

Reply:
So, there's an issue with handling that relative to the view boxes. The padding that we do have is for text and the navigation padding relies on an iteration, which is using top padding as section height padding vs subsection height padding vs menu item padding differs. This means that the interior padding needs to use a combination of padding rather than one. If you can suggest a resolution to that difficulty, I will run with it and use it. I just couldn't figure it out.

Reply:
I am not sure if we should put css classes into child component however I feel like the key is to ensure the consistency. I have been seeing paddings classes in many places. I do want to take it back thought that we can rely on child components to decide their paddings. We can talk about it more if we got the chance.

Reply:
The other thing I have noticed is that many slds classes we used in our code are not quite necessary. Some of them can inherit from parent and some of them do not make a difference. Maybe we should review it at some time.

Reply:
It's an interesting question. The main thing I am trying to preserve is an explicit callout to behavior that does not depend on a parent component. Within the same component we can get away with that, but I don't want to make assumptions on the child component's context.